SERVER/ROUTES/Routes.js

```diff
@@ -6,7 +6,7 @@ import {verifyToken} from '../JWT/jwt_conf';
 const router= express.Router();
 
 //User accounts routes
-router.post('/api/v1/auth/signup',account_controler.createUseraccount); //Endpoint to create user account
+router.post('/api/v2/auth/signup',account_controler.createUseraccount); //Endpoint to create user account
```

Comment:
you don't need unnecessary comments. Using descriptive variable names is a much better way of documenting your code than commenting.

Use comments for code that is confusing, hard to understand or read, and code that will cause a side effect if changed

Reply:
I am going to remove comments